pages/understanding-json-schema/reference/annotations.md

```diff
@@ -8,6 +8,8 @@ validation, but are used to describe parts of a schema. None of these
 \"annotation\" keywords are required, but they are encouraged for good
 practice, and can make your schema \"self-documenting\".
 
+Annotation keywords can be put inside any schema or subschema object. However, there cannot be duplicates in the same object.
```

Comment:
> 

This alternative wording looks more understandable.

Reply:
Thanks for reviewing! I have changed to the alternative wording

Reply:
I think we drop "object" and just stick with "schema".  It should be understood that it's an object as boolean schemas are just `true`/`false`.

> Annotation keywords can be used in any schema or sub-schemas.

Since schemas are inherently representable in the JSON data model, it should also be understood that duplicate keys are not permitted.  (Technically, they're permitted but different system handle them differently, so they should be avoided.)